acceptance/bundle/deploy/mlops-stacks/output.txt

```diff
@@ -0,0 +1,103 @@
+
+>>> cat config.json
+{
+  "input_project_name": "project_name_[UNIQUE_NAME]",
+  "input_root_dir": "test_repo_mlops_stacks",
+  "input_include_models_in_unity_catalog": "no",
+  "input_cloud": "[CLOUD_ENV_BASE]"
+}
+
+>>> [CLI] bundle init mlops-stacks --config-file config.json
+Welcome to MLOps Stacks. For detailed information on project generation, see the README at https://github.com/databricks/mlops-stacks/blob/main/README.md.
+
+*** Your MLOps Stack has been created in the 'test_repo_mlops_stacks/project_name_[UNIQUE_NAME]' directory! ***
+
+Please refer to the README.md for further instructions on getting started.
+
+>>> cat test_repo_mlops_stacks/README.md
+# test_repo_mlops_stacks
+
+This directory contains an ML project based on the default
+[Databricks MLOps Stacks](https://github.com/databricks/mlops-stacks),
+
+>>> [CLI] bundle summary
+Name: project_name_[UNIQUE_NAME]
+Target: dev
+Workspace:
+  User: [USERNAME]
+  Path: /Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/dev
+Resources:
+  Experiments:
+    experiment:
+      Name: /Users/[USERNAME]/[dev [USERNAME]] dev-project_name_[UNIQUE_NAME]-experiment
+      URL:  (not deployed)
+  Jobs:
+    batch_inference_job:
+      Name: [dev [USERNAME]] dev-project_name_[UNIQUE_NAME]-batch-inference-job
+      URL:  (not deployed)
+    model_training_job:
+      Name: [dev [USERNAME]] dev-project_name_[UNIQUE_NAME]-model-training-job
+      URL:  (not deployed)
+  Models:
+    model:
+      Name: [dev [USERNAME]] dev-project_name_[UNIQUE_NAME]-model
+      URL:  (not deployed)
+
+>>> [CLI] bundle validate
+Warning: expected a string value, found null
+  at targets.dev.workspace.host
```

Comment:
Are the warnings expected / relevant here?

Are they new or were always there just not visible?

Reply:
Sure, we have Badness field in test.toml to record broken that a given test exposes.

Reply:
added it to the test in the follow-up commit